subsys/bluetooth/mesh/Kconfig

```diff
@@ -1044,12 +1044,14 @@ config BT_MESH_BLOB_IO_FLASH
 
 config BT_MESH_DFU_SRV
 	bool "Support for Firmware Update Server model"
+	select BT_MESH_MODEL_EXTENSIONS
```

Comment:
Probably it is better to use `depends on` dependency, since reverse selection via `select` doesn't check `BT_MESH_MODEL_EXTENSIONS` dependencies.